app/frontend/helpers/printConversionRates.js

```diff
@@ -1,13 +1,20 @@
+// copied from https://stackoverflow.com/a/2901298
+const addCommas = (amount) => {
+  const parts = amount.toString().split('.')
+  parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+  return parts.join('.')
+}
+
 const formatFiat = (amount, maxDigits) => {
   const newAmount = (amount * 0.000001).toFixed(2)
   const amountString = newAmount.toString()
   if (amountString.length <= maxDigits) {
-    return newAmount
+    return addCommas(newAmount)
```

Comment:
wouldn't `.toLocaleString('en-US')` do the same?

Reply:
No, `'17847894984.1484'.toLocaleString('en-US') === '17847894984.1484'`.

Reply:
`parseFloat('17847894984.1484').toLocaleString('en-US')` so if you pass a number instead of a string it behaves as expected

Reply:
Wow, thank you, looks much clearer.